libs/form/src/next/input-field-masked/stories/vl-input-field-masked.stories-doc.mdx

```diff
@@ -11,7 +11,7 @@ Dit component erft over van de [input-field-next](/docs/components-next-form-inp
 ## Voorbeeld
 
 ```js
-import { VlInputFieldMaskedComponent } from '@domg-wc/components/next/form/input-field/input-field-masked';
+import { VlInputFieldMaskedComponent } from '@domg-wc/form/next/input-field/input-field-masked';
```

Comment:
Kan je de import hier vervangen door:
`'@domg-wc/form/next/input-field-masked'`
Dit ben ik blijkbaar vergeten te doen toen ik de input-field-mask heb verhuisd.

Reply:
- opzet-structuur uitgebreid met form, ineens ook qlik toegevoegd en support aangepast
- nx-workspace-setup was al in orde
- vl-input-field-masked is gecorrigeerd